src/installer/corehost/build.sh

```diff
@@ -74,10 +79,11 @@ __MsbuildDebugLogsDir="$__LogsDir/MsbuildDebugLogs"
 
 # Set the remaining variables based upon the determined build configuration
 __DistroRidLower="$(echo $__DistroRid | tr '[:upper:]' '[:lower:]')"
-__BinDir="$__RootBinDir/bin/$__DistroRidLower.$__BuildType"
+__BinDir="$__RootBinDir/bin"
```

Comment:
Should we remove `__DistroRidLower` everywhere? Why is it removed in some places, but not others?

Reply:
I do not think exported `__BinDir` is actually used. I tried to use it as the way to discover the coreclr artefacts and that forced me to normalize `__BinDir` between Unix and Windows. The Windows one was just `%__rootDir%\artifacts\bin`.

However I ended up using a separate `__CoreClrArtifacts` for few other reasons - the location of artifacts may in theory vary relatively to the bin dir. A separate name is more robust and less confusing. Extra variable will make even more sense when linking native libraries, since they would need yet another artifacts path.

`__CMakeBinDir` matters and it stayed as it was. 
`__BinDir` was made the same as on Windows, but I do not think it matters. 

I left it like this because it was confusing that Unix/Windows defined it differently. I think that is another reason why is not used - it is difficult to use in platform-neutral way.

Since it raises concerns I will revert `__BinDir` to the previous state.